haskell-load.el

```diff
@@ -88,7 +88,14 @@ Restarts the SESSION's process if that is the case."
    ((haskell-process-consume process "\nBuilding \\(.+?\\)\\.\\.\\.")
     (let ((msg (format "Building: %s" (match-string 1 buffer))))
       (haskell-interactive-mode-echo (haskell-process-session process) msg)
-      (haskell-mode-message-line msg)))))
+      (haskell-mode-message-line msg)))
+   ((string-match "\\(Collecting type info for [[:digit:]]+ module(s) ...\\)"
```

Comment:
Please remove the `\\(` and `\\)` and use `string-match 0` instead of `string-match 1` later on.